extensions/wikia/AdEngine/js/provider/netzathleten.js

```diff
@@ -0,0 +1,73 @@
+/*global define, require*/
+define('ext.wikia.adEngine.provider.netzathleten', [
+	'ext.wikia.adEngine.slotTweaker',
+	'ext.wikia.adEngine.utils.eventDispatcher',
+	'wikia.document',
+	'wikia.lazyqueue',
+	'wikia.window'
+], function (slotTweaker, eventDispatcher, doc, lazyQueue, win) {
+	'use strict';
+
+	var initialized = false,
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this useless assignment to local variable "initialized" [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AS1854)